src/Compilers/Core/Desktop/CommandLine/CommonCommandLineParser.cs

```diff
@@ -612,9 +654,9 @@ private static IEnumerable<string> Split(string str, Func<char, bool> splitHere)
         }
 
         /// <summary>
-        /// Condense double backslashes that precede a quotation mark to single backslashes.
+        /// Make all backslashes escaped regardless whether they preceed a quote or not.
```

Comment:
Is this comment accurate? It looks like backslashes preceding a quote are handled differently.